package-lock.json

```
@@ -1,15 +1,15 @@
 {
   "name": "heroicons-lookup",
-  "version": "1.1.0",
+  "version": "1.3.2",
```

```suggestion
  "version": "2.0.0",
```
Remember to change the version here too.